tests/unit/amo/reducers/test_users.js

```diff
@@ -112,6 +106,63 @@ describe(__filename, () => {
     });
   });
 
+  describe('loadUserAccount', () => {
+    it('sets notifications to `null` when loading a user', () => {
+      const user = createUserAccountResponse({ id: 12345, username: 'john' });
+      const action = loadUserAccount({ user });
+
+      const state = reducer(initialState, action);
+
+      expect(action.type).toEqual(LOAD_USER_ACCOUNT);
+      expect(action.payload).toEqual({ user });
+      expect(state.isUpdating).toEqual(false);
+      expect(state.byID[user.id]).toEqual({
+        ...user,
+        notifications: null,
+      });
+    });
+  });
+
+  describe('loadCurrentUserAccount', () => {
+    it('sets notifications to `null` when loading the current user', () => {
+      const userId = 12345;
+      const user = createUserAccountResponse({ id: userId, username: 'john' });
+
+      const state = reducer(initialState, loadCurrentUserAccount({ user }));
+
+      expect(state.byID[userId]).toEqual({
+        ...user,
+        notifications: null,
+      });
+      expect(state.currentUserID).toEqual(userId);
+    });
+  });
+
+  describe('loadUserNotifications', () => {
+    it('loads notifications for a user', () => {
+      const userId = 12345;
+      const user = createUserAccountResponse({ id: userId, username: 'john' });
+      const notifications = createUserNotificationsResponse();
+
+      const prevState = reducer(initialState, loadCurrentUserAccount({ user }));
+
+      expect(prevState.byID[userId]).toEqual({
+        ...user,
+        notifications: null,
```

Comment:
You've kind of already tested this above, in `it('sets notifications to `null` when loading the current user'`, but I guess there's no harm in asserting it again here.